pkg/cloudprovider/types.go

```diff
@@ -43,6 +43,10 @@ type Capacity interface {
 	GetArchitectures(ctx context.Context) ([]string, error)
 	// GetOperatingSystems returns the operating systems supported by the cloud provider.
 	GetOperatingSystems(ctx context.Context) ([]string, error)
+	// GetCapacityTypes returns the capacity types supported by the cloud provider.
+	GetCapacityTypes(ctx context.Context) ([]string, error)
+	// DefaultCapacityType returns the default capacity type for the cloud provider
+	DefaultCapacityType(ctx context.Context) (string, error)
```

Comment:
If we go this route of making it a first class concept, we could follow a convention of "default first" to avoid extra API calls.

Reply:
interesting, are you concerned about bloating the cloud provider interface or of making unnecessary external API calls (since in this case, the types are hardcoded) ?

Reply:
I guess I'm not 100% sure what the `DefaultCapacityType` method is needed for?